betterproto/tests/util.py

```diff
@@ -60,3 +63,25 @@ def get_test_case_json_data(test_case_name, json_file_name=None):
 
     with open(test_data_file_path) as fh:
         return fh.read()
+
+
+def find_module(
+    module: ModuleType, predicate: Callable[[ModuleType], bool]
+) -> Optional[ModuleType]:
+    if predicate(module):
+        return module
+
+    module_path = pathlib.Path(*module.__path__)
+
+    for sub in list(module_path.glob("**/")):
+        if not sub.is_dir() or sub == module_path:
+            continue
+        sub_module_path = sub.relative_to(module_path)
+        sub_module_name = ".".join(sub_module_path.parts)
+
+        sub_module = importlib.import_module(f".{sub_module_name}", module.__name__)
```

Comment:
it looks to me like this will get upset if it finds a directory without an __init__.py, maybe searching with `.glob("**/__init__.py")` then `sub_module_path = sub.parent.relative_to(module_path)` would be safer/more explicit?

Reply:
Technically, all the directories generated by betterproto will contain `__init__.py`, so it should never crash. But, if anyone were to use this method later on another directory, it will cause a problem. On the other hand, this method serves the current purpose, and not any other (for example, it cannot find modules such as `foo/bar.py`), and I'm not sure I should broaden the scope to support usecases beyond what is currently needed.

Reply:
I've changed the method to add a docstring and to search for init 👍